Vagrantfile

```diff
@@ -46,8 +46,10 @@ Vagrant.configure('2') do |cfg|
       end
 
       # Networking
-      #config.vm.network :forwarded_port, guest: 22, host: 2222 #TODO make this configurable
-
+      config_yaml['defaults']['ports'].each do |port|
```

Comment:
please change this line to the following, so one can specify forwarded ports on node level (!= global default), too. And also to make port configuration optional (``|| {}``)

```
forward_ports = get('ports', config_yaml, settings) || {}
forward_ports.each do |port|
```